core/metacling/src/TClingClassInfo.cxx

```diff
@@ -634,6 +634,33 @@ ptrdiff_t TClingClassInfo::GetBaseOffset(TClingClassInfo* base, void* address, b
    return binfo.Offset(address, isDerivedObject);
 }
 
+std::vector<std::string> TClingClassInfo::GetUsingNamespaces()
+{
+   // Find and return all 'using' declarations of namespaces.
+   std::vector<std::string> res;
+
+   R__LOCKGUARD(gInterpreterMutex);
+
+   cling::Interpreter::PushTransactionRAII RAII(fInterp);
+   const clang::DeclContext *DC = cast<DeclContext>(fDecl);
+   if (!DC)
+       return res;
+
+   clang::DeclContext::decl_iterator iter = DC->noload_decls_begin();
+   while (iter != DC->noload_decls_end()) {
```

Comment:
Perhaps you want to take a look how clang does it [here](https://github.com/root-project/root/blob/master/interpreter/llvm/src/tools/clang/lib/Sema/SemaLookup.cpp#L1196-L1215).

Reply:
This is a limited feature... Not sure how much we want to try to replicate proper lookup...